### PR TITLE
chore: clean out `core/oracle.json`

### PR DESCRIPTION
**Pull request by aljo242**

### CODE REVIEW

**Comment by aljo242** [comment]
https://slinky-sidecar-pr-401.dev.skip-internal.money/slinky/oracle/v1/prices

File: config/core/oracle.json [hunk 1]
```diff
@@ -545,81 +417,8 @@
...
-        "readTimeout": 0,
-        "writeTimeout": 0,
-        "pingInterval": 0,
-        "maxReadErrorCount": 0,
-        "maxSubscriptionsPerConnection": 0
-      },
-      "type": "price_provider"
     }
   ],
-  "production": true,
```
Comment (Eric-Warehime) [inline config/core/oracle.json:622]: Is this field still used for default log level or did we override all of that? Can't remember what the `production` field actually controls at this point.

**Review by Eric-Warehime — APPROVED** [review]
lgtm

File: config/core/oracle.json (hunk 1 above)
Comment (aljo242) [reply config/core/oracle.json:622]: its deprecated and not used anywhere

**Pull request merged**